ideas/122-sob-metrics.md

```diff
@@ -0,0 +1,72 @@
+## Preamble
+
+    Idea: 122-sob-metrics
+    Title: Open Bounty - Platform Usage Metrics
+    Status: Draft
+    Created: 2018-04-04
+    Related: 
+
+## Summary
+
+To identify the successful use of a platform, it's important and imperative to be able to measure its usage. As the Open Bounty platform continues to grow and evolve, its important to be able to track its growth, and make data-driven decisions.
+
+One of the key objectives of Open Bounty is to increase the usage of the platform. The purpose of this swarm is to build the tools and means necessary to capture these metrics and be able to report on them on a regular basis.
+
+## Swarm Participants
+
+- Lead Contributor\PM: @arash009 (Arash)
+- Contributor: @siphiuel (Vitaliy)
+- Testing and QA: 
```

Comment:
SOB tester here? or I guess we can put down swarm lead @arash009 here to not let this be a blocker